ferrocene/ci/docker-images/ubuntu-20/Dockerfile

```diff
@@ -149,3 +146,12 @@ RUN mkdir /home/ci \
     && chown -R ci: /home/ci
 USER ci
 WORKDIR /home/ci
+
+# We prefer uv over other solutions, it's in Rust and fast
+ENV PATH="/home/ci/.local/bin:$PATH"
+RUN python3 -m pip install uv
```

Comment:
Is the `pipx` in the Ubuntu 20.04 archives too old?

Reply:
It seemed so to me. I tried to use the same instructions for both and had failures. 😵